invenio_damap/assets/semantic-ui/js/invenio_damap/DMPAuthButton.js

```diff
@@ -0,0 +1,52 @@
+// This file is part of Invenio-DAMAP
+// Copyright (C) 2023-2024 Graz University of Technology.
+// Copyright (C) 2023-2024 TU Wien.
+//
+// Invenio-DAMAP is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React from "react";
+import { Button, Icon, Popup } from "semantic-ui-react";
+
+import PropTypes from "prop-types";
+
+// TODO: Make overridable
+// TODO: We could render an extra dialog instead of redirecting. This should be possible when
+// DAMAP offers a way to check between null (no identity) vs empty list (no DMPs created).
+export class DMPAuthButton extends React.Component {
+  openDAMAP = () => {
+    const { damapUrl } = this.props;
+    window.open(damapUrl, "_blank");
+  };
+
+  render() {
+    const { loading } = this.props;
+    return (
+      <div style={{ display: "flex", alignItems: "center" }}>
+        <Button
+          fluid
+          onClick={this.openDAMAP}
+          disabled={false}
+          primary
+          size="medium"
+          aria-haspopup="dialog"
+          icon
+          labelPosition="left"
+          loading={loading}
+        >
+          <Icon name="key" />
+          {"Authenticate with DAMAP"}
+        </Button>
+        <Popup
+          content="Almost there! To proceed, make sure that you've created at least one DMP in DAMAP, or check that your eligibilty requirements are fully met."
```

Comment:
minor/shelve: mark strings in front end as translatable

Reply:
I marked all the strings I could find as translatable. Can you double-check if I missed anything? :smiley:

Reply:
Reverted everything concerning translations, will open an issue and start from a clean state for this